src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,29 @@
 package core.mate.academy.model;
 
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Excavator
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String engineManufacturer;
+    private int arrowLength;
+
     public Excavator() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Excavator started to work");
     }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The `get` method is not typically a method you would override in a class like `Excavator`. This method seems to be creating instances of `Excavator` and returning them as a list, which is not a standard behavior for a class representing a single machine. Consider removing this method or ensuring it aligns with the intended design pattern.